tests/python/relay/test_pass_partition_graph.py

```diff
@@ -678,6 +678,81 @@ def expected():
     check_result(mod, {"y": y_data}, (8, 8), np.log(np_add))
 
 
+def test_multiple_outputs():
```

Comment:
Can you add a test case for something like `conv2d + batch_norm`?

Reply:
Not sure I get you; Can you elaborate more as to how multiple outputs in regions are exercised in conv2d + batch_norm?

Reply:
batch_norm has 3 outputs.

Reply:
But isnt it returned as a tuple already ? Looks like a single output to me. Unless, you do some procesing afterwards in the region. Is that what you imply?

Reply:
Oh yes. The case in my mind is `conv2d->BN->conv2d`. In this case, the region output would be a new tuple of the rest 2 BN outputs and 1 conv2d output.

Reply:
Added the testcase